consensus/ibft/src/test/java/org/hyperledger/besu/consensus/ibft/queries/IbftQueryServiceImplTest.java

```diff
@@ -48,6 +48,7 @@
 public class IbftQueryServiceImplTest {
 
   private Blockchain blockchain = mock(Blockchain.class);
```

Comment:
You could switch these two mocks over to the annotation and add the mockito runner to this test (as that is pretty consistent pattern being used throughout the tests)

Reply:
I've seen a mix of this, but will switch these over if that's the direction we're going.  👍